datastore/db/bytebase/FertiScan/schema_0.0.6.sql

```diff
@@ -0,0 +1,238 @@
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."metric" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "value" float NOT NULL,
+    "unit_id" uuid REFERENCES "fertiscan_0.0.6".unit(id),
+    "edited" boolean
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."element_compound" (
+    "id" SERIAL PRIMARY KEY,
+    "number" int NOT NULL,
+    "name_fr" text NOT NULL,
+    "name_en" text NOT NULL,
+    "symbol" text NOT NULL UNIQUE
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."label_information" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "lot_number" text,
+    "npk" text,
+    "registration_number" text,
+    "n" float,
+    "p" float,
+    "k" float,
+    "weight" uuid REFERENCES "fertiscan_0.0.6".metric(id),
+    "density" uuid REFERENCES "fertiscan_0.0.6".metric(id),
+    "volume" uuid REFERENCES "fertiscan_0.0.6".metric(id)
+    );
+
+        CREATE TABLE "fertiscan_0.0.6"."first_aid" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "first_aid_fr" text,
+    "first_aid_en" text,
+    "edited" boolean,
+    "label_id" uuid REFERENCES "fertiscan_0.0.6".label_information(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."warranty" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "warranty_fr" text,
+    "warranty_en" text,
+    "edited" boolean,
+    "label_id" uuid REFERENCES "fertiscan_0.0.6".label_information(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."instruction" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "instruction_fr" text,
+    "instruction_en" text,
+    "edited" boolean,
+    "label_id" uuid REFERENCES "fertiscan_0.0.6".label_information(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."caution" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "caution_fr" text,
+    "caution_en" text,
+    "edited" boolean,
+    "label_id" uuid REFERENCES "fertiscan_0.0.6".label_information(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."micronutrient" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "read_name" text NOT NULL,
+    "value" float NOT NULL,
+    "unit" text NOT NULL,
+    "element_id" int REFERENCES "fertiscan_0.0.6".element_compound(id),
+    "label_id" uuid REFERENCES "fertiscan_0.0.6".label_information(id),
+    "edited" boolean
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."guaranteed" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "read_name" text NOT NULL,
+    "value" float NOT NULL,
+    "unit" text NOT NULL,
+    "element_id" int REFERENCES "fertiscan_0.0.6".element_compound(id),
+    "label_id" uuid REFERENCES "fertiscan_0.0.6".label_information(id),
+    "edited" boolean
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."ingredient" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "organic" boolean NOT NULL,
+    "name" text NOT NULL,
+    "edited" boolean,
+    "label_id" uuid REFERENCES "fertiscan_0.0.6".label_information(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.6"."analysis" (
```

Comment:
Is this the "toplevel" object? By that I mean if I have this, can I get all the other objects?

Reply:
I'm not so sure what you mean. However this is only the script of the schema. 
Analysis represent everything about an analysis (samples, owner, label) so yes it is the highest level of object in the DB I'd say